machine_learning/dbscan/dbscan.ipynb

```diff
@@ -0,0 +1,318 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## DBSCAN"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
```

Comment:
Please first explain what __DBSCAN__ is, when I would use it, and why it is supercool.  Don't assume that your reader know already and try to get them excited to read more.

Reply:
On it.